index.html

```diff
@@ -26,7 +26,7 @@
         <meta property="twitter:image" content="/boltz-preview.jpg" />
         <meta
             name="description"
-            content="Boltz is a Privacy First, Non-Custodial Bitcoin Exchange and Lightning Service Provider" />
+            content="Boltz is a Non-Custodial Bitcoin Bridge, Connecting Different Layers of Bitcoin." />
```

Comment:
Why are half of the words capitalized?

Reply:
It's called "Headline Case", see https://www.enago.com/academy/best-practices-for-using-headline-case